Template/Tag/LivezillaDynamicTag.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+namespace Piwik\Plugins\TagManager\Template\Tag;
+
+use Piwik\Settings\FieldConfig;
+use Piwik\Plugins\TagManager\Template\Tag\BaseTag;
+use Piwik\Validators\CharacterLength;
+use Piwik\Validators\NotEmpty;
+use Piwik\Validators\NumberRange;
+use Piwik\Validators\UrlLike;
+
+class LivezillaDynamicTag extends BaseTag
+{
+	public function getIcon()
+    {
+        // You may optionally specify a path to an image icon URL, for example:
+        //
+        // return 'plugins/TagManager/images/MyIcon.png';
+        //
+        // to not return default icon call:
+        // return parent::getIcon();
+        //
+        // The image should have ideally a resolution of about 64x64 pixels.
+        return 'plugins/TagManager/images/icons/livezilla_icon.png';
+    }
+	
+	public function getCategory()
+    {
+        return self::CATEGORY_SOCIAL;
+    }
+
+    public function getParameters() {
+        return array(
+            $this->makeSetting('LivezillaDynamicID', '', FieldConfig::TYPE_STRING, function (FieldConfig $field) {
+                $field->title = 'Livezilla ID';
+                $field->description = 'Insert the Livezilla ID from your Dynamic Code section.';
+                $field->validators[] = new NotEmpty();
+                $field->validators[] = new CharacterLength(32);
+            }),
+			$this->makeSetting('LivezillaDynamicDomain', '', FieldConfig::TYPE_STRING, function (FieldConfig $field) {
```

Comment:
I'm not quite sure but could this be maybe automatically configured in the JavaScript depending on the current domain? eg `var LivezilleDomain = localhost.origin` or so?

Reply:
@tsteur I am not sure, but as I understood it this is the URL where Livezilla is installed (so similar to the Matomo URL)

Reply:
I see. Makes sense 👍